preview/src/widget.js

```diff
@@ -90,44 +90,77 @@ window.addEventListener('DOMContentLoaded', () => {
         .ready
         .then(onLoad);
 
-    const params = new URLSearchParams(window.location.search);
+    function renderExperience(previewMode, experience) {
+        if(previewMode === 'slotDefaultContent') {
+            document.getElementById('preview-experience')
+                .closest('li')
+                .remove();
+
+            return;
+        }
 
-    if (params.has('experience')) {
-        document.getElementById('preview-experience').textContent = params.get('experience');
+        document.getElementById('preview-experience').textContent = experience;
     }
 
-    if (params.has('audience')) {
-        document.getElementById('preview-audience').textContent = params.get('audience');
+    function renderAudience(previewMode, audience) {
+        if(previewMode === 'slotDefaultContent') {
+            document.getElementById('preview-audience').textContent = 'None';
+
+            return;
+        }
+
+        document.getElementById('preview-audience').textContent = audience;
+    }
+
+    function renderContentAndExperiment(previewMode, variant, experiment) {
```

Comment:
```suggestion
    function renderExperiment(previewMode, variant, experiment) {
```

Reply:
Keep them separed

Reply:
I'm not sure if dividing the code into different functions led to any improvement